src/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
- [ ] [OPTIONAL] Good job so far! Please move `index.html` file to `dist` folder.
- [ ]  [HINT] Use this structure:

![image](https://user-images.githubusercontent.com/63915024/181046908-e7f7919b-d73e-441d-9eaa-0e494dcf5d97.png)